runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPanel.java

```diff
@@ -257,6 +358,15 @@ private void buildBox(LootTrackerRecord record)
 		updateOverall();
 	}
 
+	void onGroupingModeChanged(boolean group)
```

Comment:
on? rename this to changeGroupingMode, onX/Y would mean this is some sort of event, but it is not